app/controllers/api/database.php

```diff
@@ -281,6 +285,300 @@
+    ->label('sdk.response.model', Response::MODEL_ATTRIBUTE_LIST)
+    ->param('collectionId', null, new UID(), 'Collection unique ID. You can create a new collection with validation rules using the Database service [server integration](/docs/server/database#createCollection).')
+    ->inject('response')
+    ->inject('dbForInternal')
+    ->action(function ($collectionId, $response, $dbForInternal) {
+        /** @var Appwrite\Utopia\Response $response */
+        /** @var Utopia\Database\Database $dbForInternal */
+
+        $collection = $dbForInternal->getCollection($collectionId);
+
+        $attributes = $collection->getAttributes();
+
+        $response->dynamic2(new Document([
+            'sum' => \count($attributes)
+            'attributes' => $attributes
+        ]), Response::MODEL_ATTRIBUTE_LIST);
+    });
+
+App::get('v1/database/collections/:collectionId/attributes/:attributeId')
+    ->desc('Get Attribute')
+    ->groups(['api', 'database'])
+    ->label('scope', 'attributes.read')
+    ->label('sdk.namespace', 'database')
+    ->label('sdk.platform', [APP_PLATFORM_SERVER])
+    ->label('sdk.method', 'listAttributes')
+    ->label('sdk.description', '/docs/references/database/get-attribute.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_ATTRIBUTE)
+    ->param('collectionId', null, new UID(), 'Collection unique ID. You can create a new collection with validation rules using the Database service [server integration](/docs/server/database#createCollection).')
+    ->param('attributeId', '', new UID(), 'Attribute unique ID.')
+    ->inject('response')
+    ->inject('dbForInternal')
+    ->action(function ($collectionId, $attributeId, $response, $dbForInternal) {
+        /** @var Appwrite\Utopia\Response $response */
+        /** @var Utopia\Database\Database $dbForInternal */
+
+        $collection = $dbForInternal->getCollection($collectionId);
+
+        $attributes = $collection->getAttributes();
+
+        $response->dynamic2($attributes[$attributeId]
+        ), Response::MODEL_ATTRIBUTE);
+    });
+
+App::delete('/v1/database/collections/:collectionId/attributes/:attributeId')
+    ->desc('Delete Attribute')
+    ->groups(['api', 'database'])
+    ->label('scope', 'attributes.write')
+    ->label('event', 'database.attributes.delete')
+    ->label('sdk.namespace', 'database')
+    ->label('sdk.platform', [APP_PLATFORM_SERVER])
+    ->label('sdk.method', 'deleteAttribute')
+    ->label('sdk.description', '/docs/references/database/delete-attribute.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
+    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->param('collectionId', null, new UID(), 'Collection unique ID. You can create a new collection with validation rules using the Database service [server integration](/docs/server/database#createCollection).')
```

Comment:
```suggestion
    ->param('collectionId', '', new UID(), 'Collection unique ID. You can create a new collection with validation rules using the Database service [server integration](/docs/server/database#createCollection).')
```

Let's have a consistent method for all default values we use for UIDs